RecoJets/Configuration/python/RecoPFJets_cff.py

```diff
@@ -61,6 +61,9 @@
     )
 
 
+fixedGridRhoFastjetCentral = fixedGridRhoFastjetAll.clone(
+    maxRapidity = cms.double(2.5)
+    )
```

Comment:
Maybe if we're going to go that route we should just put this into any "silver" miniAOD file directly?